src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/AnnotationUtils.java

```diff
@@ -50,16 +50,7 @@ public static String encodeAnyASList( final List<?> somethingList) {
      * @return true if the annotation is expected to have values per-allele
      */
     public static boolean isAlleleSpecific(final InfoFieldAnnotation annotation) {
-        if (annotation instanceof AS_RankSumTest) {
-            return true;
-        }
-        if (annotation instanceof AS_StrandBiasTest) {
-            return true;
-        }
-        if (annotation instanceof AS_RMSMappingQuality) {
-            return true;
-        }
-        if (annotation instanceof AS_StandardAnnotation) {
+        if (annotation instanceof AlleleSpecificAnnotation) {
```

Comment:
This branch references the new `AlleleSpecificAnnotation` interface in several places, but does not seem to actually introduce it. Maybe you just forgot to `git add` it?